datamodel.combodo-approval-extended.xml

```diff
@@ -87,6 +87,18 @@
           <default_value/>
           <is_null_allowed>true</is_null_allowed>
         </field>
+        <field id="target_class" xsi:type="AttributeClass">
+          <class_category>bizmodel</class_category>
+          <sql>target_class</sql>
+          <default_value>UserRequest</default_value>
+          <is_null_allowed>false</is_null_allowed>
```

Comment:
```suggestion
          <default_value/>
          <is_null_allowed>false</is_null_allowed>
```

I would not prefill this, it is up to the consumer to specify for which class it is.
Default configuration will manage the default value.